resources/5/ajax.js

```diff
@@ -0,0 +1,223 @@
+const url = "http://gamf.nhely.hu/ajax2/";
+const code = "VE22CSabc123";
+document.getElementById("code").innerHTML = "code=" + code;
+
+async function read() {
+  try {
+    let response = await fetch(url, {
+      method: 'post',
+      cache: 'no-cache',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
+      },
+      body: "code=" + code + "&op=read"
+    });
+    let data = await response.text();
+    try {
+      data = JSON.parse(data);
+    } catch (jsonError) {
+      console.error("Hiba: A válasz nem JSON formátumú!", data);
```

Comment:
Console log messages should be in English, it is for the developers not for the Users